### PR TITLE
Update documentation to add Databricks 12.2 as a supported platform [skip ci]

### PR DESCRIPTION
**Pull request by andygrove**
Closes https://github.com/NVIDIA/spark-rapids/issues/8660

Update documentation to add Databricks 12.2 information

### CODE REVIEW

File: docs/additional-functionality/delta-lake-support.md [hunk 1]
```diff
@@ -49,7 +50,8 @@ operation which is typically triggered via the DataFrame `write` API, e.g.:
 `data.write.format("delta").save(...)`.
 
 Table creation from selection, table insertion from SQL, and table merges are not currently
-GPU accelerated. These operations will fallback to the CPU.
+GPU accelerated. These operations will fallback to the CPU. Writes against tables that have
+deletion vectors enabled will also fallback to the CPU.
```
[the remark this answers is not in the document]
Comment (andygrove) [reply docs/additional-functionality/delta-lake-support.md:55]: Yes. I have added this now.

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

**Comment by sameerz** [comment]
@viadea @nvliyuan @SurajAralihalli please update https://nvidia.github.io/spark-rapids/docs/FAQ.html#which-distributions-are-supported to indicate support for Databricks 12.2.

**Comment by jlowe** [comment]
I recommend pointing the FAQ to the release notes on the Downloads page which lists the supported releases.  That way we don't have to try to keep these things in sync in multiple places.  Note that they already aren't consistent, with the FAQ saying we directly support things that the release notes do not.

**Comment by SurajAralihalli** [comment]
noted!

**Pull request merged**